### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by Batarian711** [jonschlinkert/randomatic#17]
I am a graphic designer. I have a logo proposal for you. Do you have an idea? What do you say?

**Comment by jonschlinkert** [comment]
Yes, that would be great! I'd love to see a logo if you're still interested in creating one! I can't believe I missed this issue somehow, apologies!
